src/notebook/plugin.ts

```diff
@@ -179,6 +194,42 @@ class NotebookFileHandler extends AbstractFileHandler {
     return ['.ipynb']
   }
 
+  executeSelectedCell(): void {
+    for (let w of this.widgets) {
+      if (w.hasClass('jp-mod-focus')) {
```

Comment:
Another option is to move `activeWidget` into `AbstractFileHandler` and let the document manager set it.